src/Native/Runtime/inc/eetype.h

```diff
@@ -444,6 +444,11 @@ class EEType
     bool IsGeneric()
         { return (m_usFlags & IsGenericFlag) != 0; }
 
+#if defined(CORERT)
+    ModuleManager* GetModule()
```

Comment:
Do we really want to stick with ModuleManager? I was thinking about proposing to rename it to Module at some point, because ModuleManager makes me think of something that manages modules (this happened on multiple occasions before I got used to it). This thing represents a module, it doesn't manage them...